CI/Dockerfile

```diff
@@ -26,31 +26,29 @@ SHELL ["/bin/bash", "-c"]
 ENV TZ=America/Chicago
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
-# Update core packages
-RUN apt-get -y update; \
-    apt-get -y install autoconf \
-                       clang \
-                       cmake \
-                       g++ \
-                       gcc \
-                       gfortran \
-                       libhdf5-dev \
-                       libtool \
-                       libeigen3-dev\
-                       python3-numpy \
-                       python3 \
-                       python3-pip \
-                       python3-setuptools \
-                       python3-dev \
-                       libpython3-dev \
-                       wget \
-                       software-properties-common; \
-    add-apt-repository ppa:git-core/ppa; \
-    apt-get update; \
-    apt-get install -y git; \
-    update-alternatives --install /usr/bin/python python /usr/bin/python3 10; \
-    update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 10; \
-    pip install "cython<3";
+# Install Build Dependencies
+RUN apt-get update --yes && \
+    apt-get install --yes \
+    git \
+    wget \
+    autoconf \
+    clang \
+    cmake \
+    g++ \
+    gcc \
+    gfortran \
+    libhdf5-dev \
+    libtool \
+    libeigen3-dev\
+    python3-numpy \
+    python3 \
+    python3-pip \
+    python3-setuptools \
+    python3-dev \
+    libpython3-dev \
+    software-properties-common && \
```

Comment:
I prefer these to be indented a little more to make it clear that they are arguments to `apt-get` and not additional commands to be RUN